### DISCUSSION TITLE
Problem with mastodon installation in debian 12 VM

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#26853] [Q&A]
Hello, let me introduce myself, my name is David, I am encountering a problem during the installation and finalization of the mastodon instance.

![Capture d'écran 2023-09-08 134747](https://github.com/mastodon/mastodon/assets/22295927/52ef274e-b8f6-4a9b-bd02-ede5928ba0d0)

So would you be so kind as to help me? 
- I am unable to configure nginx and SSL certificates.
- I would like to finally be able to finalize the installation on my localhost.

I thank you in advance for your help. I hope to have been clear and for simplicity, I will give access to my VM.

Well thanks

**Comment by ryanmio** [comment]
For Nginx, I would make sure you've correctly copied the Mastodon Nginx template into your Nginx sites-available and symlinked it to sites-enabled. For SSL on localhost, you might consider using a self-signed certificate just to get things rolling.
